_data/libs_map.yml

```diff
@@ -324,6 +324,7 @@
   pmempool_rm: pmempool_rm.3
   pmempool_sync: pmempool_sync.3
   pmempool_transform: pmempool_sync.3
+  libpmemset: libpmemset.7
```

Comment:
I am not sure if this is generated properly (libpmemset probably should starts with -).
I will merge this anyway because of other files, but @kswiecicki please check if this man mapge works on pmem.io.